### PR TITLE
Added Marble DAO CW20 from JUNO

### PR DESCRIPTION
**Pull request by the0developer**

### CODE REVIEW

File: osmosis-1/osmosis-1.assetlist.json [hunk 1]
```diff
@@ -1214,6 +1214,37 @@
...
+      "name": "Marble",
+      "display": "marble",
+      "symbol": "MARBLE",
+      "ibc": {
+        "source_channel": "channel-47",
+        "dst_channel": "channel-169",
+        "source_denom": "umarble"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/marble-logo.png"
```
Comment (IbrarMakaveli) [inline osmosis-1/osmosis-1.assetlist.json:1244]: `        "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/marble.png"
`
Also can you rename image -> marble.png

**Review by IbrarMakaveli — CHANGES_REQUESTED** [review]

**Review by dylanschultzie — APPROVED** [review]

**Review by the0developer — COMMENTED** [review]
@IbrarMakaveli Done, it should be good. Thanks for taking the time

File: osmosis-1/osmosis-1.assetlist.json [hunk 2]
```diff
@@ -1214,6 +1214,37 @@
...
+      "description": "The native token cw20 for Marble DAO on Juno Chain",
+      "type_asset": "cw20",
+      "address": "juno1g2g7ucurum66d42g8k5twk34yegdq8c82858gz0tq2fc75zy7khssgnhjl",
+      "denom_units": [
+        {
+          "denom": "ibc/F6B691D5F7126579DDC87357B09D653B47FDCE0A3383FF33C8D8B544FE29A8A6",
+          "exponent": 0,
+          "aliases": [
+            "marble"
+          ]
```
Comment (sunnya97) [inline osmosis-1/osmosis-1.assetlist.json:1227]:
```suggestion
          "exponent": 0
```

**Comment by sunnya97** [comment]
Please add back coingecko after price is served from coingecko

**Comment by the0developer** [comment]
> Please add back coingecko after price is served from coingecko

I'll do it Sunny, thanks.

**Review by IbrarMakaveli — APPROVED** [review]

**Pull request merged**